scripts/o3de/ExportScripts/export_source_built_project.py

```diff
@@ -0,0 +1,236 @@
+#
+# Copyright (c) Contributors to the Open 3D Engine Project.
+# For complete copyright and license terms please see the LICENSE at the root of this distribution.
+#
+# SPDX-License-Identifier: Apache-2.0 OR MIT
+#
+#
+
+import argparse
+import fnmatch
+import glob
+import logging
+import os
+import sys
+
+import o3de.export_project as exp
+import pathlib
+import shutil
+
+from o3de.export_project import process_command
+from o3de import manifest
+from typing import List
+
+# This script is meant to be run via o3de cli export-project command, not from Python directly!
+# This export script supports optional building of asset tools, so it is only compatible with the source distribution
+# of O3DE (https://github.com/o3de/o3de)
+
```

Comment:
should we update these instructions to include specific examples for passing in custom CMake args?